boards: nrf9280pdk: Add revision 0.2.0

bjda opened the pull request:
Add revision support and overlays for rev. 0.2.0.

bjda committed:
boards: nrf9280pdk: Add revision 0.2.0

Add revision support and overlays for rev. 0.2.0.

Signed-off-by: Bernt Johan Damslora <bernt.damslora@nordicsemi.no>

tokangas reviewed (approved):
Looks good to me! Also tried a few Zephyr samples with an nRF9280 0.2.0 PDK, all worked fine.

bjda committed:
boards: nrf9280pdk: disable IEEE 802.15.4 support

nRF92 series is not supported in the IEEE 802.15.4 stack yet,
so remove it from the board for now.

Signed-off-by: Bernt Johan Damslora <bernt.damslora@nordicsemi.no>
